network-api/networkapi/wagtailpages/customblocks/youtube_regret_block.py

```diff
@@ -1,9 +1,24 @@
 from wagtail.core import blocks
+from wagtail.images.blocks import ImageChooserBlock
 
 
 class YoutubeRegretBlock(blocks.StructBlock):
-    heading = blocks.CharBlock(
-        help_text='Temporary youtube regret block.'
+    headline = blocks.CharBlock(
+        help_text='Headline of this YouTube Regret'
+    )
+
+    image = ImageChooserBlock(
+        required=False
+    )
+
+    imageAltText = blocks.CharBlock(
+        required=False,
+        help_text='Image description (for screen readers).'
+    )
+
+    story = blocks.TextBlock(
```

Comment:
All regret stories we have are plain text.

Reply:
ah, no need for richtext then. Nice